save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsProfileMenuView.kt

```diff
@@ -2,21 +2,40 @@ package com.saveourtool.save.frontend.components.views.usersettings
 
 import com.saveourtool.save.frontend.components.basic.cardComponent
 import com.saveourtool.save.frontend.components.inputform.InputTypes
+import com.saveourtool.save.frontend.components.modal.displayModal
+import com.saveourtool.save.frontend.components.modal.mediumTransparentModalStyle
+import com.saveourtool.save.frontend.utils.*
 
 import react.VFC
-import react.dom.html.ReactHTML.button
 import react.dom.html.ReactHTML.div
 import react.dom.html.ReactHTML.hr
 import react.dom.html.ReactHTML.input
 import web.cssom.ClassName
-import web.html.ButtonType
 import web.html.InputType
 
 @Suppress("MISSING_KDOC_TOP_LEVEL")
 class UserSettingsProfileMenuView : UserSettingsView() {
     private val card = cardComponent(isBordered = false, hasBg = true)
     @Suppress("TOO_LONG_FUNCTION", "LongMethod", "EMPTY_BLOCK_STRUCTURE_ERROR")
     override fun renderMenu(): VFC = VFC {
+        val deleteUserWindowOpenness = useWindowOpenness()
+
+        displayModal(
```

Comment:
`window.confirm()`?

Reply:
I want there to be a big, visual window here and people would think what they are doing :)